src/shared_modules/utils/socketWrapper.hpp

```diff
@@ -685,7 +685,7 @@ class Socket final : public T
                 if (ret <= 0)
                 {
                     m_unsentPacketList.emplace(m_sendDataBuffer.data() + amountSent, bufferSize - amountSent);
-                    throw std::runtime_error {"Error sending data to socket: " + std::to_string(errno) + " " +
+                    throw std::runtime_error {"Error sending data to socket: " + std::string(std::strerror(errno)) + " " +
                                               std::string(strerror(errno))};
```

Comment:
Repeated error string
```suggestion
                    throw std::runtime_error {"Error sending data to socket: " + std::string(std::strerror(errno))};
```